### PR TITLE
Concurrent plotting

### PR DESCRIPTION
**Pull request by matthiasreisser**
Gave non-unique temporary directory for index.html. Now several processes can have their own plots concurrently without interference. Also, tmp dir is deleted at program exit. Test is updated to not run setup_web_plotting() twice in case web-backend has been set in config.

### CODE REVIEW

**Comment by matthiasreisser** [comment]
tmp dir is created within the artemis directory under /.artemis/tmp/web_backend/{rand_uuid4}/

File: artemis/plotting/test_plotting_server.py [hunk 1]
```diff
@@ -9,12 +10,16 @@
...
+    config = get_artemis_config()
+    if config.get('plotting', 'backend') != 'matplotlib-web':
+        setup_web_plotting()
+
+    # for i in xrange(5):
+    while True:
         dbplot(np.random.randn(10, 10, 3), 'noise')
         dbplot(np.random.randn(20, 2), 'lines')
-        plt.pause(.01)
+        plt.pause(2.)
```
Comment (petered) [inline artemis/plotting/test_plotting_server.py]: Is this test ever going to terminate?

Comment (matthiasreisser) [inline artemis/plotting/test_plotting_server.py]: No

**Comment by petered** [comment]
(run py.test)

**Comment by matthiasreisser** [comment]
2 tests fail at import tabulate. No idea...

**Comment by petered** [comment]
```
===================================================================== 81 passed, 3 skipped in 122.42 seconds ======================================================================

```

**Pull request merged**